src/index.tsx

```diff
@@ -1,22 +1,24 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+
+import ReactDOM from 'react-dom/client';
 import 'src/index.css';
 import App from 'src/App';
 import reportWebVitals from 'src/reportWebVitals';
 import { StoreProvider } from './context/context';
 import 'materialize-css/dist/css/materialize.min.css';
 import 'materialize-css/dist/js/materialize.min';
 
-ReactDOM.render(
-    <React.StrictMode>
+const root = ReactDOM.createRoot(
+    document.getElementById('root') as HTMLElement
+);
+root.render(
+   // <React.StrictMode>
```

Comment:
Needs to comment since the `react-materialize` modals cannot be opened with the current react version.

Reply:
We can create a task to address this issue specifically